### ISSUE TITLE
Unable to install with rubocop 0.79.0

### ISSUE THREAD

**Issue by nikitasakov** [mbj/mutant#992]
Hello 👋

Rubocop `0.79.0` requires parser `>= 2.7.0.1`, but mutant requires `~> 2.6.5`. So looks like it is vital to upgrade parser for mutant. 🙂

```
Fetching gem metadata from https://rubygems.org/........
Resolving dependencies...
Bundler could not find compatible versions for gem "parser":
  In snapshot (Gemfile.lock):
    parser (= 2.7.0.2)

  In Gemfile:
    mutant-rspec (= 0.9.4) was resolved to 0.9.4, which depends on
      mutant (~> 0.9.4) was resolved to 0.9.4, which depends on
        parser (~> 2.6.5)

    datarockets-style was resolved to 0.7.0, which depends on
      rubocop-rspec (~> 1.36) was resolved to 1.37.1, which depends on
        rubocop (>= 0.68.1) was resolved to 0.79.0, which depends on
          parser (>= 2.7.0.1)

Running `bundle update` will rebuild your snapshot from scratch, using only
the gems in your Gemfile, which may resolve the conflict.
```

Thanks.

**Comment by mbj** [comment]
Correct. It mostly depends on https://github.com/mbj/unparser/issues/129

**Comment by mbj** [comment]
@nikitasakov The fix was released with mutant 0.9.5.

Note that mutant at this point does not support 2.7 syntax.
